setup/Swix/Directory.Build.targets

```diff
@@ -1,10 +1,22 @@
 <Project>
 
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
   <PropertyGroup>
     <!-- for everything else, $(PackageOutputPath) is interpreted as a directory, but for the SWIX toolset a file name -->
     <PackageOutputPath>$(ArtifactsDir)\VSSetup\$(Configuration)\Insertion\$(OutputName).vsix</PackageOutputPath>
   </PropertyGroup>
```

Comment:
This is odd.

Reply:
It's the recognized way to include upstream `Directory.Build.*` files.

Reply:
Hmm, off by a few lines again. I meant 
```
<PackageOutputPath>$(ArtifactsDir)\VSSetup\$(Configuration)\Insertion\$(OutputName).vsix</PackageOutputPath>
```

Are you looking at this using CodeFlow?